### PR TITLE
Add vcs-browser and contribute URL type

### PR DESCRIPTION
**Pull request by JakobDev**
This PR adds vcs-browser and contribute as URL types as in #374. It was easier and cleaner to make a new PR than to update the old one.

### CODE REVIEW

File: src/as-enums.h [hunk 1]
```diff
@@ -93,6 +95,8 @@ typedef enum {
 	AS_URL_KIND_DONATION,
 	AS_URL_KIND_TRANSLATE,
 	AS_URL_KIND_CONTACT,
+	AS_URL_KIND_VCS,
```
Comment (ximion) [inline src/as-enums.h:98]: This should be `AS_URL_KIND_VCS_BROWSER` shouldn't it?

**Review by ximion — CHANGES_REQUESTED** [review]
Looks good to me, except for the VCS enum name that should go after its string representation, for clarity.

**Comment by JakobDev** [comment]
I had now replaced AS_URL_KIND_VCS with AS_URL_KIND_VCS_BROWSER

**Review by ximion — APPROVED** [review]

**Comment by ximion** [comment]
Looks good now, thank you for the patch! :-)

**Comment by JakobDev** [comment]
I informed [GNOME](https://gitlab.gnome.org/GNOME/gnome-software/-/issues/1702) and [KDE](https://bugs.kde.org/show_bug.cgi?id=452054) that the new types has been added. Would be nice, if you can create a new release. It will take some time until the distros ship the new Version, so the Implementation in the Software Centers can be started.

**Comment by cassidyjames** [comment]
Thanks @JakobDev, I'll file the appropriate issues on elementary software.

**Pull request merged**